build-consumer.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+ls -l
+ls -l ../
+git clone ssh://git@github.com/Shopify/$1
+cd $1
+ls -l
+yarn
```

Comment:
yarn supports a `--cwd=` option, and git supports a `-C` option to specify the current working directory, so you can get rid of the cd above, and I believe doing a `yarn add` implies installing anything else that is missing as well, so I think this can be slightly shortened:

```
git clone -C=..ssh://git@github.com/Shopify/$1
yarn --cwd=../$1 add file:../polaris-react
yarn --cwd=../$1 build
```